metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -108,26 +144,163 @@ def __init__(self, inspector_instance: Inspector):
         # tables that we don't want to ingest into the DataHub
         self.exclude_tablespaces: Tuple[str, str] = ("SYSTEM", "SYSAUX")
 
+    def has_table(self, table_name, schema=None):
```

Comment:
DataHub uses a few selective methods from inspector interface. You can look them up [here](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py) . Also listed below for quick reference. I believe it should suffice to implement only these, as required.

```
get_schema_names
get_table_names
get_view_names
get_pk_constraint
get_table_comment
get_view_definition
get_columns
get_foreign_keys
```

I doubt whether other methods implemented in wrapper here are needed.
These methods are not used by Datahub and therefore should be removed.
```
has_table
has_sequence
_resolve_synonym
_prepare_reflection_args
 get_temp_table_names
get_table_options
get_indexes
get_check_constraints
```

Reply:
Unused methods were deleted.